recipes/r-hydenet/meta.yaml

```diff
@@ -0,0 +1,101 @@
+{% set version = '0.10.11' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-hydenet
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:     {{ cran_mirror }}/src/contrib/Archive/HydeNet/HydeNet_{{ version }}.tar.gz
+  sha256: 33c530d7edb49ba3b3fe5324f23b8a0ae9799281faa26f835af8c7dcc43c28e2
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-diagrammer >=0.9.0
+    - r-checkmate
+    - r-dplyr
+    - r-graph
```

Comment:
This is a no-go. The CRAN package `graph` was moved to Bioconductor more than a decade ago. This is available on Bioconda as `bioconductor-graph`.